### PR TITLE
Mgcb relative path fix when calling with /@:

### PR DESCRIPTION
**Pull request by cra0zy**
#4190

### CODE REVIEW

File: Tools/MGCB/CommandLineParser.cs [hunk 1]
```diff
@@ -284,6 +284,8 @@ private IEnumerable<string> Preprocess(IEnumerable<string> args)
                 if (arg.StartsWith("/@:") || arg.StartsWith("-@:"))
                 {
                     var file = arg.Substring(3);
+                    Directory.SetCurrentDirectory(Path.GetDirectoryName(file));
```
Comment (tomspilman) [inline Tools/MGCB/CommandLineParser.cs]: I'm nervous at this sort of change.  It has the potential to break every other file operation.

First why was this not a problem before?  We've been using MGCB on Mac/Linux for months now...  at least a year on Windows.  What is different about your case and what happens to all the previously working cases?

Comment (cra0zy) [inline Tools/MGCB/CommandLineParser.cs]: Pipeline sets working directory in: https://github.com/mono/MonoGame/blob/develop/Tools/Pipeline/Common/PipelineController.cs#L475 so it wasn't noticeable with it.

Plus it's not like this is changing much with this, working directory can still be overwritten with "workingDir" argument.

Also I've actually known about this for about 9 months or so... http://community.monogame.net/t/starting-directory-for-mgcb-when-loading-response-file-wrong/1608 I just forgot about it till now....

Comment (KonajuGames) [inline Tools/MGCB/CommandLineParser.cs]: I added the workingDir argument a while back so I could debug from MonoDevelop, which has no working directory option in its debug settings.

The usual mode of operation when running MGCB from the command line has been to change directory to the location of the .mgcb file, but setting the current directory from the location of the .mgcb file would make the debugging process easier in MonoDevelop (no need for the workingDir argument).

Comment (jamesford42) [inline Tools/MGCB/CommandLineParser.cs]: I think the correct fix is to expand all path's to absolute when the preprocessor expands a response file. This fixes your issue without messing with the working directory for anyone downstream. Also note that restoring the working directory after the response file could work, so long as it is done in a stack so response files with response files don't break.

**Pull request closed without merging**